### ISSUE TITLE
Chart text not accessible - low contrast

### ISSUE THREAD

**Issue by AWearring** [localgovdrupal/localgov_elections#147]
The contrast levels of text on each row of the charts do not conform to the required 4.5:1 ratio to meet WCAG 2.2 AA

https://demo.localgovdrupal.org/election/general-election-july-2024

![Image](https://github.com/user-attachments/assets/17e1989c-140e-4bcc-8ae7-1485a8e7fffc)

https://demo.localgovdrupal.org/election/general-election-july-2024/banbury

![Image](https://github.com/user-attachments/assets/bf68931c-7b75-43a0-a4e4-7009eceb6efc)

**Comment by markconroy** [comment]
Closing in favour of https://github.com/localgovdrupal/localgov_elections/issues/178
